### ISSUE TITLE
How to get Real time market data from KS API for trading? Seems that KS doesn't have websocket.

### ISSUE THREAD

**Issue by abhishektrivedi333** [osparamatrix/ks-orderapi-python#26]

**Comment by tupe-aniket** [comment]
checkout ks.quote api

**Comment by kalilinux-png** [comment]
what kind of real time date you want do you want ask price and bid price or open high low close prices

**Comment by aquibchiniwala** [comment]
Is there a way to get live ltp or depth by using websocket or something else?

**Comment by aquibchiniwala** [comment]
@booooozaa quote api doesn't have a websocket interface to get live ticker updates

**Comment by kalilinux-png** [comment]
@aquibchiniwala to get the live data try the following code 
while True:
     print(client.quote(instrument_code,'LTP'))

the above code will give you the live trading price with very little latency

**Comment by aquibchiniwala** [comment]
Yes but it will block a large amount of CPU. Any idea when will the Websocket be implemented as it is a basic requirements for any trading apis. It would be great if it is developed on a very high priority bases.

**Comment by kalilinux-png** [comment]
@aquibchiniwala I have no idea when it will be implement i think you should contact developers of this api

**Comment by aquibchiniwala** [comment]
> @aquibchiniwala to get the live data try the following code
> while True:
> print(client.quote(instrument_code,'LTP'))
> 
> the above code will give you the live trading price with very little latency

The quote method inside a loop returns the following error after 3 successful API calls though the quota is set to 60 calls/Minute.

HTTP response body: {"fault":{"code":900807,"message":"Request quota reached","description":"You have surpassed the maximum number of requests for the minute, please wait until the next minute to start additional requests"}}

@kalilinux-png  do you have a solution for this? 

Also I am not able to update the quota settings in the API Manager.

**Comment by kalilinux-png** [comment]
@aquibchiniwala brother while adding api to your app you might have selected wrong please select the policy as unlimited 
if you still not able to get rid of this error :
please watch the video given below from 1: 50 - 2: 50 
please let me know if this does not solve you issue
